lib/Parser.js

```diff
@@ -37,6 +38,107 @@ const POSSIBLE_AST_OPTIONS = [{
 	}
 }];
 
+class StackedSetMap {
```

Comment:
If not, add a //TODO so I can come around and add an issue for later for other contribs to handle.